python/ray/__init__.py

```diff
@@ -12,7 +12,7 @@
 
 from ray.worker import (register_class, error_info, init, connect, disconnect,
                         get, put, wait, remote, log_event, log_span,
-                        flush_log, get_gpu_ids)  # noqa: E402
+                        flush_log, get_gpu_ids, get_webui_url)  # noqa: E402
```

Comment:
Linting says this is imported but unused. The way to deal with this is to add it to the list `__all__` below.